packages/yew/src/server_renderer.rs

```diff
@@ -39,7 +40,20 @@ where
 {
     /// Creates a [ServerRenderer] with custom properties.
     pub fn with_props(props: ICOMP::Properties) -> Self {
-        Self { props }
+        Self {
+            props,
+            hydratable: true,
+        }
+    }
+
+    /// Sets whether an the rendered result is hydratable.
+    ///
+    /// Defaults to `true`.
+    ///
+    /// When this is sets to `true`, the rendered artifact will include assistive nodes
+    /// to assist with the hydration process.
```

Comment:
```suggestion
    /// When this is sets to `true`, the rendered artifact will include comment nodes
    /// to assist with the hydration process.
```
to be more specific and not duplicate assistive.